existing_LSTM_model/machine_learning_instruction/analysis_generator_1/report/GeneratorReportGeneratedSequences.csv

```diff
@@ -0,0 +1,11 @@
+Generated Sequences
```

Comment:
one thing about exporting data: try to use a standard format, e.g., AIRR, where the column name for this would be `sequence_aa`, and never put spaces in column names -> many potential issues down the line :D